### PR TITLE
chore: add support for Kubernetes v1.19.15 and v1.20.11 on Azure Stack

### PR DESCRIPTION
**Pull request by haofan-ms**
**Reason for Change**:

**Issue Fixed**:

**Credit Where Due**:

Does this change contain code from or inspired by another project?

- [ ] No
- [ ] Yes

If "Yes," did you notify that project's maintainers and provide attribution?

- [ ] No
- [ ] Yes

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

**Comment by haofan-ms** [comment]
Looks like I need to wait until @mboersma 's PRs for 1.19.15 and 1.20.11 support to merge and pull master again to pass the test issues, since I didn't include 1.19.15 and 1.20.11 in the AllKubernetesSupportedVersions map.

File: vhd/packer/configure-windows-vhd.ps1 [hunk 1]
```diff
@@ -117,9 +117,8 @@ function Get-FilesToCacheOnVHD {
             "https://kubernetesartifacts.azureedge.net/csi-proxy/v0.2.2/binaries/csi-proxy-v0.2.2.tar.gz"
         );
         "c:\akse-cache\win-k8s\"      = @(
-            "https://kubernetesartifacts.azureedge.net/kubernetes/v1.18.18-azs/windowszip/v1.18.18-azs-1int.zip",
-            "https://kubernetesartifacts.azureedge.net/kubernetes/v1.19.10-azs/windowszip/v1.19.10-azs-1int.zip",
-            "https://kubernetesartifacts.azureedge.net/kubernetes/v1.20.6-azs/windowszip/v1.20.6-azs-1int.zip",
+            "https://kubernetesartifacts.azureedge.net/kubernetes/v1.19.10-azs/windowszip/v1.19.15-azs-1int.zip",
```
Comment (jadarsie) [inline vhd/packer/configure-windows-vhd.ps1:120]: bad URIs

**Review by jadarsie — CHANGES_REQUESTED** [review]

**Review by jadarsie — DISMISSED** [review]

**Review by mboersma — APPROVED** [review]
/lgtm

Thanks for your patience @haofan-ms!

**Review by jackfrancis — APPROVED** [review]
/lgtm

**Pull request merged**